src/components/browse/datasetlist/components/datasetDetailsLayout.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
```

Comment:
The component is the top level folder name. Do not use components further down the structure. Follow the same structure we follow inside the components for nesting. Browse is one of the components and it contains sidebar and datasetList. So datasetList will have its own file and folders and so on..

Reply:
Done